### PR TITLE
openbsd: add directory; openbsd/pkg_{add,info,delete}: add page

### PR DESCRIPTION
**Pull request by mrusme**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

As [mentioned here](https://github.com/tldr-pages/tldr/issues/816#issuecomment-1196781520) I'm creating what seem to be the first few tldrs for OpenBSD, that include `pkg_add`, `pkg_info` and `pkg_delete`.

Closes https://github.com/tldr-pages/tldr/issues/816

### CODE REVIEW

File: pages/openbsd/pkg_add.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# pkg_add
+
+> OpenBSD package manager utility.
+> Search for packages using `pkg_info`, remove packages using `pkg_delete`.
```
Comment (kbdharun) [inline pages/openbsd/pkg_add.md:4]:
```suggestion
> See also: `pkg_info`, `pkg_delete`.
```

File: pages/openbsd/pkg_add.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+`pkg_add -u`
+
+- Install a new package:
+
+`pkg_add {{package}}`
+
+- Install packages from the raw output of `pkg_info`:
+
+`pkg_add -l {{file}}`
```
Comment (kbdharun) [inline pages/openbsd/pkg_add.md:17]:
```suggestion
`pkg_add -l {{path/to/file}}`
```

File: pages/openbsd/pkg_add.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+# pkg_add
+
+> OpenBSD package manager utility.
```
Comment (kbdharun) [inline pages/openbsd/pkg_add.md:3]:
```suggestion
> Install/update packages in OpenBSD.
```

File: pages/openbsd/pkg_delete.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+# pkg_delete
+
+> OpenBSD package manager utility.
+> Add packages using `pkg_add`, search for packages using `pkg_info`.
```
Comment (kbdharun) [inline pages/openbsd/pkg_delete.md:4]:
```suggestion
> Remove packages in OpenBSD.
> See also: `pkg_add`, `pkg_info`.
```

File: pages/openbsd/pkg_delete.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+`pkg_delete {{package}}`
+
+- Delete package, including unused dependencies:
+
+`pkg_delete -a {{package}}`
+
+- Dry-run deletion of package:
+
+`pkg_delete -n {{package}}`
```
Comment (kbdharun) [inline pages/openbsd/pkg_delete.md:17]:
```suggestion
- Delete a package:

`pkg_delete {{package}}`

- Delete a package, including it's unused dependencies:

`pkg_delete -a {{package}}`

- Dry-run deletion of a package:

`pkg_delete -n {{package}}`
```

File: pages/openbsd/pkg_info.md [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# pkg_info
+
+> OpenBSD package manager utility.
+> Add packages using `pkg_add`, remove packages using `pkg_delete`.
```
Comment (kbdharun) [inline pages/openbsd/pkg_info.md:4]:
```suggestion
> View information about packages in OpenBSD.
> See also: `pkg_add`, `pkg_delete`.
```

File: pages/openbsd/pkg_info.md [hunk 7]
```diff
@@ -0,0 +1,13 @@
...
+> More information: <https://man.openbsd.org/pkg_info>.
+
+- Search for package using package name:
```
Comment (kbdharun) [inline pages/openbsd/pkg_info.md:7]:
```suggestion
- Search for a package using the package name:
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have started a discussion in our chatroom regarding the new directory addition, let's hope we reach a consensus on it soon.

I have a few suggestions now. I will add a general `pkg` page which refers to these subpages.

**Review by kbdharun — APPROVED** [review]

**Comment by kbdharun** [comment]
Hi, I have taken over this PR and performed the changes as specified in the maintainer's guide; Thanks for your initial merge.

For those who haven't checked the chatroom, other maintainers approved having a new openbsd directory for its unique platform-specific commands and packages.

Post this PRs merge, I will update the Client specification to add support for this directory.

File: pages/openbsd/pkg_delete.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+
+> Remove packages in OpenBSD.
+> See also: `pkg_add`, `pkg_info`.
+> More information: <https://man.openbsd.org/pkg_delete>.
+
+- Delete a package:
+
+`pkg_delete {{package}}`
+
+- Delete a package, including it's unused dependencies:
```
Comment (acuteenvy) [inline pages/openbsd/pkg_delete.md:11]:
```suggestion
- Delete a package, including its unused dependencies:
```

File: pages/openbsd/pkg_info.md [hunk 9]
```diff
@@ -0,0 +1,13 @@
...
+
+> View information about packages in OpenBSD.
+> See also: `pkg_add`, `pkg_delete`.
+> More information: <https://man.openbsd.org/pkg_info>.
+
+- Search for a package using the package name:
+
+`pkg_info -Q {{package}}`
+
+- Output list of installed packages for use with `pkg_add -l {{list}}`:
```
Comment (acuteenvy) [inline pages/openbsd/pkg_info.md:11]:
```suggestion
- Output a list of installed packages for use with `pkg_add -l`:
```

**Review by acuteenvy — APPROVED** [review]

**Review by gutjuri — APPROVED** [review]

**Comment by kbdharun** [comment]
Just now went through the client specification, I don't think we need to add any additional information about the change as https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#platform and https://github.com/tldr-pages/tldr/blob/main/CLIENT-SPECIFICATION.md#directory-structure are already written in a generalized manner.

**Comment by acuteenvy** [comment]
Clients still have to be updated though.

**Comment by kbdharun** [comment]
> Clients still have to be updated though.

Yeah. In the next client specification release, we can mention the new platform addition in the meanwhile we can focus on documenting its pages (maybe with a Let's document issue). Btw, what do you think we should version the next release of client specification? (the last one was 2.0 since it introduced a major change).

**Comment by acuteenvy** [comment]
There's an ongoing client spec PR (#10730) so maybe we can add it there (and bump the version too).

**Comment by kbdharun** [comment]
> There's an ongoing client spec PR (#10730) so maybe we can add it there (and bump the version too).

Yep that's exactly what I am inquiring about, what version number should we choose for the future release so that we can create a milestone (here on this repo) to track all things related to it before creating a release.

**Comment by kbdharun** [comment]
> Also, if we are going to add all the BSD platforms, it would be great to add all of them (freebsd, netbsd, etc.) in one PR so as not to change the spec every other day.

I think we are only going to add `openbsd`/`freebsd` pages to the new directory. Kinda curious, does `netbsd` have a lot of unique commands?

**Comment by acuteenvy** [comment]
> what version number should we choose for the future release

Do we consider the placeholder escaping syntax a breaking change? If so, I guess it should be `3.0`. Otherwise `2.1`.

> does `netbsd` have a lot of unique commands?

A package manager, that's for sure. There's even an [issue](https://github.com/tldr-pages/tldr/issues/7077) to document it. Other than that, I don't know, I've never used NetBSD.

**Comment by kbdharun** [comment]
> Do we consider the placeholder escaping syntax a breaking change? If so, I guess it should be 3.0. Otherwise 2.1.

IMO it isn't a breaking change so we can go with a minor version `v2.1`. Will create the version now and tag both the PRs.

> A package manager, that's for sure. There's even an https://github.com/tldr-pages/tldr/issues/7077 to document it. Other than that, I don't know, I've never used NetBSD.

We could document it if there are multiple commands, if not then we can include them under openbsd itself but mention it is for NetBSD.

**Pull request merged**